ufraw_exiv2.cc:343:10: note: 'std::cerr' is defined in header '<iostream>'
ufraw_exiv2.cc:365:14: error: 'cerr' is not a member of 'std'

--- a/ufraw_exiv2.cc
+++ b/ufraw_exiv2.cc
@@ -17,6 +17,7 @@
 #ifdef HAVE_EXIV2
 #include <exiv2/exiv2.hpp>
 #include <sstream>
+#include <iostream>
 #include <cassert>
 
 /*
